VPC peering article on GCP (#1408)
aiven/devportal: docs/platform/howto/vpc-peering-gcp.rst

.. _vpc-peering-gcp:

> Set Virtual Private Cloud (VPC) peering on Google Cloud Platform (GCP)

Reviewer: ```suggestion
Set up Virtual Private Cloud (VPC) peering on Google Cloud Platform (GCP)
```

======================================================================

> Once you've created a :ref:`VPC on the Aiven platform <platform_howto_setup_vpc_peering>`, you can follow this instruction to set up VPC peering on AWS.

Reviewer: ```suggestion
Once you've created a :ref:`VPC on the Aiven platform <platform_howto_setup_vpc_peering>`, you can follow these instructions to set up VPC peering on GCP.
```

1. Open your GCP Console.

> 2. Select **VPC Networks** on the left and find the VPC that you want to connect.

Reviewer: ```suggestion
2. Select **VPC Networks** and find the VPC that you want to connect to.
```

Reviewer: I would suggest removing "on the left" from here, if reasonable, to avoid needing to check/update things later if GCP changes their console UI.

3. Click the project name and make note of the *Project ID*.

4. Under *VPC Networks*, make note of the VPC Network Name.

5. In the Aiven web console, select the VPC connection that you created.

6. Enter the project ID and VPC network name from GCP, then click **Add peering connection**.
This adds a new connection with the *Pending Peer* status.

7. In your GCP Console, go to **VPC** > **VPC network peering** and select **Create Connection**.

8. Enter a name for the peering connection and enter the same project ID and network name that are shown in the Aiven web console.

   .. note::

>        You can click the blue Pending peer icon to see the Aiven project name and the network name.

Reviewer: Super minor point, but in step 6 Pending Peer uses title case. :)

>        You can click the blue Pending peer icon to see the Aiven project name and the network name.

Reviewer: Would this be better as step 7? The same way we guide the users to make note of the GCP info in steps 3 and 4, we can tell them to get the project ID and network name before creating the connection in GCP. What do you think?
